types/braintree/braintree-tests.ts

```diff
@@ -23,7 +34,8 @@ const gateway: BraintreeGateway = new braintree.BraintreeGateway({
         customerId: '123456',
         streetAddress: '222 Oak Street',
     };
-    const response = await gateway.address.create(addressRequest);
+    const response = await gateway.address.create(addressRequest).catch(console.error);
+    if (!response) return;
```

Comment:
Yeah, I suppose this is a good idea for other contributors' envs where they might not have a gateway setup.